core/templates/dev/head/galleries/gallery.html

```diff
@@ -16,7 +16,7 @@
   <div ng-controller="SearchBar">
     <div class="pull-right oppia-navbar-button-container">
       <!-- NOTE: If you change the links here, please make the corresponding change in the 'no results' message. -->
-      {% if not user_email %}
+      {% if not has_user_registered_as_editor %}
```

Comment:
It would be worth adding a comment here that this will force the sign-up page to show up when a user accesses the gallery again, if they need to agree to the terms of use. Also, will this also work when creating an exploration, playing an exploration, etc? Ie, will I be forced to agree to the new terms of use again in those situations?